From b886a7db3c6669ccf1e69747cca2fdddc54c15ee Mon Sep 17 00:00:00 2001
From: Rafael Laverde <leafar91@gmail.com>
Date: Wed, 12 Jul 2017 19:20:06 -0500
Subject: [PATCH] Set encoding when opening languages files, fix error on
 windows.

---
 lektor/i18n.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/lektor/i18n.py b/lektor/i18n.py
index 8f7f977..5947057 100644
--- a/lektor/i18n.py
+++ b/lektor/i18n.py
@@ -1,5 +1,6 @@
 import os
 import json
+from io import open
 
 from lektor._compat import iteritems
 from lektor.uilink import UI_LANG
@@ -13,7 +14,8 @@
 
 translations = {}
 for _lang in KNOWN_LANGUAGES:
-    with open(os.path.join(translations_path, _lang + '.json')) as f:
+    with open(os.path.join(translations_path, _lang + '.json'),
+              encoding="utf8") as f:
         translations[_lang] = json.load(f)
 
 
